reconciler/testing/tracker.go

```diff
@@ -39,6 +39,9 @@ var _ tracker.Interface = (*FakeTracker)(nil)
 // OnChanged implements OnChanged.
 func (*FakeTracker) OnChanged(interface{}) {}
 
+// OnDeletedParent implements OnDeletedParent.
+func (*FakeTracker) OnDeletedParent(interface{}) {}
```

Comment:
should this delete from `n.references`?

Reply:
Good point, it likely should. I'll rearrange things.